src/server/multi_test.cc

```diff
@@ -782,4 +782,77 @@ TEST_F(MultiEvalTest, MultiSomeEval) {
   EXPECT_THAT(brpop_resp, ArgType(RespExpr::NIL_ARRAY));
 }
 
+TEST_F(MultiEvalTest, MultiPullsEarly) {
```

Comment:
to change to another test

Reply:
Didn't quite understand what pulls early - it looks just like a regular multi consistency/isolation test

Reply:
renamed and added one-liner. This test reproduces a bug that Adi solved but struggled to reproduce in C++.